types/types.d.ts

```diff
@@ -1484,10 +1484,11 @@ export interface Page {
   context(): BrowserContext;
 
   /**
-   * Browser-specific Coverage implementation, only available for Chromium atm. See
-   * [ChromiumCoverage](#class-chromiumcoverage) for more details.
+   * > NOTE: Only available for Chromium atm.
```

Comment:
Can we rephrase to not mention "Automated Teller Machine"? 😄

Reply:
I'd rather not focus on it atm :P